### PR TITLE
Update README with standalone hint

### PR DESCRIPTION
**Pull request by matteodem**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ play('MyButton', module)
   })
 ```
 
-You can also put the example component in a seperate file, like `.vue` file and register components there, locally.
+You can also put the example component in a seperate file, like `.vue` file and register components there, locally. Don't forget to use the `--standalone` option if you're using vue-play-cli.
```
Comment (egoist) [inline README.md]: um, maybe change to:

> if you're using vue-play-cli and template option in component.

Comment (matteodem) [reply README.md]: The description is beneath the code example with the template.

Comment (egoist) [reply README.md]: so why not add this there?

Comment (matteodem) [reply README.md]: sure, done

**Comment by matteodem** [comment]
should be good now right? 🙂

**Comment by egoist** [comment]
I tweaked it a bit. 😉

**Comment by matteodem** [comment]
Cool! really happy so far with it

**Pull request merged**